execution/executor/tests/db_bootstrapper_test.rs

```diff
@@ -276,10 +274,6 @@ fn test_pre_genesis() {
     let (db, db_rw) = DbReaderWriter::wrap(LibraDB::new_for_test(&tmp_dir));
     restore_state_to_db(&db, accounts_backup, proof, root_hash, PRE_GENESIS_VERSION);
 
-    // DB is not empty, `bootstrap_db_if_empty()` won't apply default genesis txn.
-    assert!(bootstrap_db_if_empty::<LibraVM>(&db_rw, &genesis_txn)
-        .unwrap()
-        .is_none());
```

Comment:
test your bootstrap_genesis() called with old waypoint returns error on this?

Reply:
this is actually not an old waypoint, it's a wrong waypoint, I can assert it returns an error.
both waypoint are at version 0 but with different state.

Reply:
yes that's what I meant.